libs/SmartStore/SmartStore/Classes/SFStoreCursor.m

```diff
@@ -92,22 +90,6 @@ - (NSString*)getDataSerialized:(SFSmartStore*)store error:(NSError**)error {
     [resultBuilder appendFormat:@"\"%@\":", @"currentPageOrderedEntries"];
     [store queryAsString:resultBuilder querySpec:self.querySpec pageIndex:[self.currentPageIndex integerValue] error:error];
     [resultBuilder appendString:@"}"];
-    //Verify the entire string for JSON format
```

Comment:
The issue seems to really be with externally stored files being incomplete, the code to combine json strings into the complete result does need the check.